libs/core/langchain_core/utils/function_calling.py

```diff
@@ -374,6 +374,8 @@ def convert_to_openai_function(
 
 def convert_to_openai_tool(
     tool: Union[Dict[str, Any], Type[BaseModel], Callable, BaseTool],
+    *,
+    strict: Optional[bool] = None,
```

Comment:
Do we want to add this in core, or just glom it onto the dict in `ChatOpenAI`?

Reply:
imo core, suspect other apis will follow

Reply:
also this function is defined as returning oai format so seems fair game